lib/services.identity.js

```diff
@@ -376,6 +432,60 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
       404: 'The identity was not found on the system.'
     }
   });
+
+  app.delete(idPath, ensureAuthenticated, function(req, res, next) {
+    var fullUrl = req.protocol + '://' + req.get('host') + req.originalUrl;
```

Comment:
This doesn't look like the right way to do this. cc: @davidlehn -- what's the right fix here?